vmdk_plugin/utils/config/config_windows.go

```diff
@@ -24,4 +24,8 @@ var (
 	DefaultConfigPath = filepath.Join(os.Getenv("PROGRAMDATA"), "docker-volume-vsphere", "docker-volume-vsphere.conf")
 	// DefaultLogPath is the default location of the log (trace) file.
 	DefaultLogPath = filepath.Join(os.Getenv("LOCALAPPDATA"), "docker-volume-vsphere", "logs", "docker-volume-vsphere.log")
+	// DefaultSharedPluginConfigPath is the default location of configuration file for shared plugin
+	DefaultSharedPluginConfigPath = filepath.Join(os.Getenv("PROGRAMDATA"), "docker-volume-shared", "docker-volume-shared.conf")
+	// DefaultSharedPluginLogPath is the default location of log (trace) file for shared plugin
+	DefaultSharedPluginLogPath = filepath.Join(os.Getenv("LOCALAPPDATA"), "docker-volume-shared", "logs", "docker-volume-shared.log")
```

Comment:
If all goes well we will look into supporting  it on Windows in v2, though it will use native SMB I'd guess.